test/lib/game/round_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule RoundTest do
+  use ExUnit.Case, async: true
+
+  test "start_link/1 takes users and sets state" do
+    users = ["Tim", "Eli"]
+    {:ok, pid} = Reactor.Game.Round.start_link(users)
+    assert is_pid(pid)
```

Comment:
In all seriousness, I want to test something like "it correctly sets the initial state". I think it would be so much easier if the return value was like {:ok, pid, state}